### DISCUSSION TITLE
Careful specifying calib_mode within .yaml file

### DISCUSSION THREAD

**Discussion by valeriabriones** [ua-snap/dvm-dos-tem#684] [General]
When digging into the config.js files testing a calibration run with nfeeed=True and nfeed=False, I noticed that the runs where nfeed was supposed to be False, calib_mode: GPPALLIgnoringNitrogen was actually true. It seems by default in the ca-.yaml file the 'ALL' is capitalized when in MADSTEMDriver.py the calib_mode is written as 'GPPAllIgnoringNitrogen'. Thus be careful this is the way in which calib_mode is set otherwise nfeed with remain True

**Comment by tobeycarman** [comment]
Oh good catch! I updated the demo configs and made the code a little more flexible in commit fe6e23285.
Closing discussion for now. Can re-open if more bugs come up.
